cmd/kueuectl/app/options/update_workload_status_options.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/kueue/client-go/clientset/versioned"
+	"sigs.k8s.io/kueue/client-go/clientset/versioned/scheme"
+	kueuev1beta1 "sigs.k8s.io/kueue/client-go/clientset/versioned/typed/kueue/v1beta1"
+	"sigs.k8s.io/kueue/cmd/kueuectl/app/util"
+)
+
+type UpdateWorkloadStatusOptions struct {
+	PrintFlags *genericclioptions.PrintFlags
+
+	DryRunStrategy   util.DryRunStrategy
+	Name             string
+	Namespace        string
+	EnforceNamespace bool
+
+	Client kueuev1beta1.KueueV1beta1Interface
+
+	PrintObj printers.ResourcePrinterFunc
+
+	genericiooptions.IOStreams
+}
+
+func NewUpdateWorkloadStatusOptions(streams genericiooptions.IOStreams, operation string) *UpdateWorkloadStatusOptions {
+	return &UpdateWorkloadStatusOptions{
+		PrintFlags: genericclioptions.NewPrintFlags(operation).WithTypeSetter(scheme.Scheme),
```

Comment:
What's the outcome of passing the operation to `NewPrintFlags`? Where will it show up?

Reply:
On result we will have `workload.kueue.x-k8s.io/wl1 resumed` and `workload.kueue.x-k8s.io/wl1 stopped`. Also we can output result on json and yaml if its need with `-o` parameter. And of course `workload.kueue.x-k8s.io/wl1 resumed (server dry run)` and `workload.kueue.x-k8s.io/wl1 resumed (client dry run)` for dry-run operations.

Reply:
I see, thanks!